index.js

```diff
@@ -1,10 +1,15 @@
 'use strict';
+
+if (process.type === 'renderer') {
+	throw new Error('Cannot use electron-context-menu in renderer process!');
+}
```

Comment:
I think it should only throw if it's actually called.

Reply:
You mean only if contextMenu is being created in the renderer process (this check will be moved to export function)?

Reply:
Yes